mcs_benchmark_data/pipelines/commonsense_qa/commonsense_qa_benchmark_transformer.py

```diff
@@ -21,7 +21,8 @@ def _transform_benchmark_sample(
     ) -> Generator[_Model, None, None]:
 
         sample_jsonl_file_path = self._sample_jsonl_file_path(
-            dataset_type=dataset_type, content_type=ContentType.SAMPLES.value
+            dataset_type=dataset_type,
+            dataset_content_type=DatasetContentType.SAMPLES.value,
```

Comment:
Use the enum itself, not its .value.

Reply:
The method declaration has the enum, not its .value, as the type annotation of this parameter.